apps/transaction/tests/test_forms/test_transaction_create_form_on_commit.py

```
@@ -0,0 +1,113 @@
+        called exactly once by form.save(), without handle_message being called
+        directly in the same stack frame.
+        """
+        user = UserFactory()
+        other_user = UserFactory()
+        room = RoomFactory(created_by=user)
+        room.users.add(user, other_user)
+        currency = CurrencyFactory()
+
+        on_commit_call_count = []
+
+        def spy_on_commit(func, *args, **kwargs):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `spy_on_commit`

Add return type annotation: `None`

(ANN202)